### PR TITLE
Switch to kubevirtci/nfs-ganesha

### PR DESCRIPTION
**Pull request by rmohr**
Host the image in kubevirtci to restore the build since
slintes/nfs-ganesha is deleted.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -378,17 +378,17 @@ container_pull(
 # Pull nfs-server image
 container_pull(
     name = "nfs-server",
-    digest = "sha256:2a8c1cad9156165d0be1ee2c78fb2342f8368193acff72eb276ff276512508fa",
+    digest = "sha256:8c1fa882dddb2885c4152e9ce632c466f4b8dce29339455e9b6bfe71f0a3d3ef",
     registry = "index.docker.io",
-    repository = "slintes/nfs-ganesha",  # see https://github.com/slintes/docker-nfs-ganesha
+    repository = "kubevirtci/nfs-ganesha",  # see https://github.com/slintes/docker-nfs-ganesha
```
Comment (oshoval) [inline WORKSPACE:383]: slintes in the comment ?

**Comment by dhiller** [comment]
/release-note-none
--

File: WORKSPACE (hunk 1 above)
Comment (rmohr) [reply WORKSPACE:383]: Yep, the official repo.

**Comment by rmohr** [comment]
/hold

verifying it now locally and will merge asap.

File: WORKSPACE (hunk 1 above)
Comment (dhiller) [reply WORKSPACE:383]: no, i think this still applies. @rmohr you haven't cloned the repo, have you?

**Comment by rmohr** [comment]
/hold cancel

Working

```
• [SLOW TEST:127.170 seconds]
Storage
/root/go/src/kubevirt.io/kubevirt/tests/storage_test.go:54
  Starting a VirtualMachineInstance
  /root/go/src/kubevirt.io/kubevirt/tests/storage_test.go:64
    [rfe_id:3106][crit:medium][vendor:cnv-qe@redhat.com][level:component]With ephemeral alpine PVC
    /root/go/src/kubevirt.io/kubevirt/tests/storage_test.go:249
      should be successfully started
      /root/go/src/kubevirt.io/kubevirt/vendor/github.com/onsi/ginkgo/extensions/table/table.go:92
        with Ephemeral PVC from NFS
        /root/go/src/kubevirt.io/kubevirt/vendor/github.com/onsi/ginkgo/extensions/table/table_entry.go:41
```

**Pull request merged**